tools/depends/target/libdvdread/Makefile

```diff
@@ -16,14 +16,21 @@ else
   ifeq ($(PLATFORM),)
     PLATFORM = native
     TARBALLS_LOCATION = $(ROOT_DIR)
-    PKGCONFIGPATH = PKG_CONFIG_PATH="$(ROOT_DIR)/../libdvdcss/dvdcss-install/lib/pkgconfig"
-    EXTRA_CFLAGS += -I$(ROOT_DIR)/../libdvdcss/dvdcss-install/include
+    ifneq ($(BUILD_DVDCSS),0)
```

Comment:
I tested again on osx and linux and if `BUILD_DVDCSS`is not set, it will build with with libdvdcss.